executor/aggfuncs/aggfuncs.go

```diff
@@ -23,9 +23,18 @@ import (
 
 // All the AggFunc implementations are listed here for navigation.
 var (
-// All the AggFunc implementations for "COUNT" are listed here.
-// All the AggFunc implementations for "SUM" are listed here.
-// All the AggFunc implementations for "AVG" are listed here.
+	// All the AggFunc implementations for "COUNT" are listed here.
+	// All the AggFunc implementations for "SUM" are listed here.
+	// All the AggFunc implementations for "AVG" are listed here.
+	_ AggFunc = (*avgOriginal4Decimal)(nil)
```

Comment:
Why name it Original?

Reply:
means this aggregate function is in the `partial 1` or `complete` mode and it's input is the data from the child of logical aggregate operator, not the partial result of other aggregate function.

Reply:
How about `avg4Decimal`?

Reply:
`avg4Decimal` maybe confusing with `avgPartial4Decimal`, I prefer the old name.